src/views/event.hbs

```diff
@@ -1,25 +1,26 @@
-<div class="">
-  <div class="">
-    <div class="">
-      <h1>{{this.event.eventTitle}}</h1>
-      <p>{{this.event.eventDescription}}</p>
-    </div>
-    <div class="">
-      <a href="#">RSVP</a>
-      <span>{{this.event.eventDate}}</span>
-      <span>{{this.event.eventTime}}</span>
-    </div>
+<div class="cf ph6 ph6-ns pv6 ml5">
+  <div class="fl-ns w-80-ns pr4-ns">
+    <h1 class="mb3 mt0 lh-title mid-gray">{{ this.event.eventTitle}}</h1>
```

Comment:
so travis is failing cause this has changed, what do i mean is in the tests its like this :https://github.com/UKITH/Arabs-Startups/blob/master/tests/server/index.js#L143 
so in this PR change the test to be without the h1 tag